hydromt_wflow/workflows/soilgrids.py

```diff
@@ -557,9 +558,9 @@ def soilgrids(
     else:
         thetas = average_soillayers(thetas_sl, ds["soilthickness"])
     thetas = thetas.raster.reproject_like(ds_like, method="average")
-    ds_out["thetaS"] = thetas.astype(np.float32)
+    ds_out["soil_theta_s"] = thetas.astype(np.float32)
```

Comment:
if a hydromt name exists, better to use it at this stage (check naming.py)
```suggestion
    ds_out["theta_s"] = thetas.astype(np.float32)
```